Dockerfile

```diff
@@ -1,6 +1,9 @@
 # ======= FRONT-END BUILD ==========
 FROM node:20-alpine as build
 
+# Install pnpm
+RUN npm install -g pnpm@9.4.0
```

Comment:
@marcusschiesser Using a variable for pnpm version is better but it's not a big deal, we will get an error when locking the package then we will upgrade it here too.